src/features/generic_events/aggregate/user-actions/user-actions-aggregator.js

```diff
@@ -48,20 +49,18 @@ export class UserActionsAggregator {
  * Generates a selector path for the event, starting with simple cases like window or document and getting more complex for dom-tree traversals as needed.
  * Will return a random selector path value if no other path can be determined, to force the aggregator to skip aggregation for this event.
  * @param {Event} evt
- * @returns {string}
+ * @param {Array<string>} [targetFields=[]] specifies which fields to gather from the nearest element in the path
+ * @returns {{ selectorPath: (undefined|string), nearestTargetFields: {}, hasActLink: boolean}}
  */
 function getSelectorPath (evt, targetFields) {
```

Comment:
we should maybe consider renaming `getSelectorPath` and `gatherSelectorPathInfo` to something more informative of what it is doing, since it is returning a lot more than the selector path these days